fixtures/codity_prod_smoke.py

```
@@ -0,0 +1,6 @@
+def run_formula(formula: str, row: dict) -> object:
+    return eval(formula, {}, row)
+
+
+def search_user(email: str) -> str:
+    return f"select * from users where email = '{email}'"
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)